### DISCUSSION TITLE
Test if field exist in JSON

### DISCUSSION THREAD

**Discussion by sobiso** [hyperjumptech/grule-rule-engine#296] [General]
Im using something like
```
dataContext.AddJSON("MSG", jsonData)
```

How I can chein by the rule if the field for example "field1" exist in this JSON?

**Comment by newm4n** [comment]
If you want to check if a field is exist in a fact, just simply access that fact in the when, if the fact exist, it will be evaluated. If not, the then section will not be executed. If you want to have something more sophisticated, you can add an object with your custom function into the context and use that function to work with your json.
